rapids-cmake/cython/detail/skbuild_patches.cmake

```diff
@@ -0,0 +1,55 @@
+# =============================================================================
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+# =============================================================================
+
+include_guard(GLOBAL)
+
+#[=======================================================================[.rst:
+_set_python_extension_symbol_visibility
+---------------------------------------
+
+.. versionadded:: v22.06.00
+
+The original version of this function in scikit-build runs a linker script to
+modify the visibility of symbols. This version is a patch to avoid overwriting
+the visibility of symbols because otherwise any library that exports symbols
+with external linkage will have the visibility of those symbols changed
+undesirably. We can remove this function once this issue is resolved in
+scikit-build.
+
+Issue: https://github.com/scikit-build/scikit-build/issues/668
+PR: https://github.com/scikit-build/scikit-build/pull/703
+
+#]=======================================================================]
+function(_set_python_extension_symbol_visibility _target)
+  # TODO: Should we guard this function's existence based on a scikit-build version?
```

Comment:
I think we should create an issue on rapids-cmake that tracks that this needs to be guarded once scikit-build has been fixed. Otherwise for now we need to do this unconditionally since we don't know what version will have the fix.